electron/ironfish/IronFishManager.ts

```diff
@@ -236,4 +248,16 @@ export class IronFishManager implements IIronfishManager {
 
     return Promise.resolve(result)
   }
+
+  getNodeConfig(): Promise<Partial<ConfigOptions>> {
+    return Promise.resolve(this.nodeSettings.getConfig())
+  }
```

Comment:
question: Is it convention to have every method return a promise?

Reply:
As I understand this is done because we use electron `ipcRenderer` methods to access backend functions and `ipcRenderer` calls returns promise and to make demo data manager works the same way we need to wrap things in promises.
So for that case even is we don't wrap in promise it will return promise on ui side, but that allow as to use same interface for demo data.
@ndemidovich-sc please correct me if I'm wrong

Reply:
Ah, gotcha. If there's a technical reason for this, that's totally fine 👍

Reply:
It is done for use similar data types in:
- `IronfishDataManager`
- `preload` script to configure `contextBridge` (It's provide possibility to use similar Manager structure in `window` context)
- `DemoDataManager` (Possibly need reimplement it uses interfaces similar as in `IronfishManager`)